### PR TITLE
fix: fix pypi publish workflow build

### PR DESCRIPTION
**Pull request by UsamaSadiq**
## Description
- Fixed the failing pypi upgrade action build.
- Tested by triggering the custom check on the pull_request.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -82,8 +82,10 @@ def is_requirement(line):
 
 setup(
     name='concept-xblock',
-    version='0.3.0',
+    version='0.3.2',
     description='concept XBlock',   # TODO: write a better description.
+    long_description='This is a simple XBlock which will allows one to tag problems with concepts.',
```
Comment (iamsobanjaved) [inline setup.py:87]: We can instead just add `readme.rst` content to `long_description`, ref: https://github.com/openedx/edx-ora2/blob/eb9e2ac68b254f11df1da74908c21d62cc5850b0/setup.py#L62-L63

Comment (UsamaSadiq) [reply setup.py:87]: We don't have the README and version changes in this repo yet so I opted for simple approach.

File: setup.py [hunk 2]
```diff
@@ -82,8 +82,10 @@ def is_requirement(line):
 
 setup(
     name='concept-xblock',
-    version='0.3.0',
+    version='0.3.2',
     description='concept XBlock',   # TODO: write a better description.
+    long_description=open('README.md', encoding='utf-8').read(),  # pylint: disable=consider-using-with
+    long_description_content_type='text/x-rst',
```
Comment (iamsobanjaved) [inline setup.py:88]:
```suggestion
    long_description_content_type='text/markdown',
```

**Review by iamsobanjaved — APPROVED** [review]
Just one change needed

**Pull request merged**